android/app/src/main/java/com/androidstartupperfapp/react/ReactFragmentInstanceManager.java

```diff
@@ -0,0 +1,50 @@
+package com.androidstartupperfapp.react;
+
+import androidx.annotation.IdRes;
+import androidx.fragment.app.FragmentManager;
+
+import com.androidstartupperfapp.R;
+import com.facebook.react.ReactFragment;
+
+public final class ReactFragmentInstanceManager {
+  private static ReactFragment reactFragment;
+  private static FragmentManager fragmentManager;
+  @IdRes
+  private final static int reactFragmentId = R.id.react_fragment;
+  private final static String componentName = "AndroidStartupPerfApp";
+
+  public static void setFragmentManager(FragmentManager fragmentManager) {
+    ReactFragmentInstanceManager.fragmentManager = fragmentManager;
+  }
+
+  public static ReactFragment getReactFragment() {
+    return reactFragment;
+  }
+
+  public static void loadReactFragment() {
+    reactFragment = new ReactFragment.Builder()
+      .setComponentName(componentName)
+      .build();
+
+    fragmentManager
+      .beginTransaction()
+      .add(reactFragmentId, reactFragment)
+      .addToBackStack(componentName)
+      .hide(reactFragment)
```

Comment:
## Attempt 1, Part A

Present the React fragment so that we can see the behavior for what it is.

```suggestion
```